### DISCUSSION TITLE
No hotspot and no URL backend

### DISCUSSION THREAD

**Discussion by tobivan** [outdoorbits/little-backup-box#159] [Q&A] [answered]
Hi there,
first of all thanks a lot for your great work!
I have installed latest Raspi Lite 32bit via the Raspi Imager, set an SSH login and added WiFi settings for my LAN. Then I ran the command `curl -sSL https://raw.githubusercontent.com/outdoorbits/little-backup-box/main/install-little-backup-box.sh | bash`
Raspi started over again. Now the Raspi has set up two wlan: wlan0 and wlan1 both within the DHCP-range of my LAN. No hotspot showing up, and I can not get any response when entering either the one or the other IP in my webbrowser. It does show the "Apache2 Debian Default Page" when entering http://IP
Doing a port-scan on the first IP following ports are open: 21, 22, 80, 139, 445, 8200
Doing a port-scan on the MiniDLNA-IP following ports are open: 80, 139, 445, 8200
Connecting to the MiniDLNA-IP with port 8200 it shows "MiniDLNA settings" page with "Media library" and "Connected clients"

But all in all: no hotspot, no "backend" on URL.
Thanks for helping me!
Tobias

**Comment by outdoorbits** [comment, marked as answer]
Strange. I tried to reproduce this. Since 64bit is out, all my 
installations have been based on Raspberry Pi OS lite 64bit. But right 
now I followed your (expected) pathway:

1. used Raspberry Pi Imager v1.7.2 and configured it to activate ssh and 
setup access to my wlan there
2. wrote Raspberry Pi OS lite 32bit to my card
3. booted my Pi, ssh pi@xxx.xxx.xxx.xxx,  curl -sSL 
https://raw.githubusercontent.com/outdoorbits/little-backup-box/main/install-little-backup-box.sh 
| bash
4. selected the default backup mode: 4 (shouldn't make any difference)
5. Install mejiro: Yes (shouldn't make any difference)
6. Install comitup: YES (essential for you, of course)
7. info-screen seconds before the installation finishes: OK
8. waited for reboot
9. connected my laptop to the wlan of the Pi (ssid: 
little-backup-box-nnnn)
10. opened the gui at https://10.41.0.1 (could also access any other 
service, as described in the info-screen at 7.)

My installation just finished successfully.

sudo ifconfig showed wlan0 only.

Did you do any different decisions?

**Comment by tobivan** [comment]
Thanks a lot. I have tried with a different SD card and now it works perfectly. Great software!

**Comment by outdoorbits** [comment]
Thank you!
